src/python/py/models/builder.py

```diff
@@ -1855,6 +2004,10 @@ def make_model(self, input_path):
             extra_kwargs = {"num_hidden_layers": self.num_layers} if "num_hidden_layers" in self.extra_options else {}
             model = AutoModelForCausalLM.from_pretrained(self.model_name_or_path, cache_dir=self.cache_dir, token=self.hf_token, trust_remote_code=True, **extra_kwargs)
 
+        if "adapter_path" in self.extra_options:
+            from peft import PeftModel
+            model = PeftModel.from_pretrained(model, self.extra_options["adapter_path"])
```

Comment:
Why are the `cache_dir`, `token`, `trust_remote_code`, and `extra_kwargs` parameters are omitted?